src/workerd/server/tests/python/BUILD.bazel

```diff
@@ -1,59 +1,67 @@
 load("//:build/wd_test.bzl", "wd_test")
 
+load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
+
+copy_file(
+    name = "pyodide-dev.capnp.bin@rule",
+    src = "//src/pyodide:pyodide.capnp.bin",
+    out = "pyodide-bundle-cache/pyodide-dev.capnp.bin"
+)
+
 wd_test(
     src = "hello/hello.wd-test",
-    args = ["--experimental"],
+    args = ["--experimental", "--pyodide-bundle-disk-cache-dir", "$(location pyodide-dev.capnp.bin@rule)/.."],
```

Comment:
Might be time to define a `py_wd_test` and avoid repeating this

Reply:
Good idea. I was debating adding this to wd_test but your suggestion is way cleaner

Reply:
seems to still be using wd_test here